terraform.tfvars.j2

```diff
@@ -1,6 +1,6 @@
 assume_role           = "administrator"
 k2hb_version          = "0.0.180"
-al2_hardened_ami_id   = "ami-07f0879b525d97930"
+al2_hardened_ami_id   = "ami-0acc76b7cfeb54499"
```

Comment:
The AMI to use is ami-0160d5db0a2ec3a7d. This ami is an untested JVM ami. Also, we set this using TF_VAR so may not be required any more?